src/dumpulator/modules.py

```diff
@@ -95,6 +113,19 @@ def find(self, key: Union[str, int]) -> Optional[Module]:
             return self.find(base)
         raise TypeError()
 
+    def parse_forwards(self):

```

Comment:
I’m not sure about the approach. Perhaps it would be better to just store the forwarded module, export pair and deal with it during import resolution.